addons/survey/models/survey_question.py

```diff
@@ -747,6 +771,17 @@ def _check_question_not_empty(self):
             if not bool(label.question_id) != bool(label.matrix_question_id):
                 raise ValidationError(_("A label must be attached to only one question."))
 
+    @api.constrains('value', 'value_image')
```

Comment:
Actually, could we make this a SQL constraint instead? It is faster and more robust.
Too bad for the pretty error message. Note that if we weren't in Survey and we could open/close many questions without saving to the DB, this method would still have been useful to check records before they are really saved...

BTW, I did check that `required` will have been "not null", so no need for a migration :partying_face: